drupal/sites/default/boinc/modules/boincwork/includes/projectprefs.xsd

```diff
@@ -134,10 +134,10 @@
 
     <!-- grouping types -->
 
-    <xs:element name="apps">
+    <xs:element name="app_overrides">
         <xs:complexType>
             <xs:sequence>
-                <xs:element name="title" type="xs:string" minOccurs="1" maxOccurs="1" />
+                <xs:element name="title" type="xs:string" minOccurs="0" maxOccurs="1" />
```

Comment:
Hm, we could think about removing the `title` entirely. Now that it gets a [fixed value](https://github.com/BOINC/boinc/pull/2025/files#diff-ea2a15c2242d752fb293111403e676afR214) anyway there's really a point to define it anymore, let alone override it. Also, this is a string that should match the upstream value to facilitate translation which is another reason to remove the override for good. Thoughts?